### ISSUE TITLE
lf.layer_norm_grad_fn, lf.layer_norm should have 'normalized_shape' in its argument, and variance_epsilon argument name should be 'eps'

### ISSUE THREAD

**Issue by Sangyu-Han** [rachtibat/LRP-for-Transformers#22]
This is torch.nn.functional layer_norm source code.

[[docs]](https://pytorch.org/docs/stable/generated/torch.nn.functional.layer_norm.html#torch.nn.functional.layer_norm)def layer_norm(
    input: Tensor,
    normalized_shape: List[int],
    weight: Optional[Tensor] = None,
    bias: Optional[Tensor] = None,
    eps: float = 1e-5,
) -> Tensor:
    r"""Apply Layer Normalization for last certain number of dimensions.

    See :class:`~torch.nn.LayerNorm` for details.
    """
    if has_torch_function_variadic(input, weight, bias):
        return handle_torch_function(
            layer_norm,
            (input, weight, bias),
            input,
            normalized_shape,
            weight=weight,
            bias=bias,
            eps=eps,
        )
    return torch.layer_norm(
        input, normalized_shape, weight, bias, eps, torch.backends.cudnn.enabled
    )

While torch.layer_norm takes 'normalized_shape' argument(even it is not used), lf.layer_norm does not take 'normalized_shape' argument. This cause problem when we want to replace torch.nn.functional.layer_norm to lf.layer_norm. The different epsilon name(varience_epsilon in lf, eps in F.layernorm) also have similar problem

As nn.functional.layer_norm node in torch.fx takes node.args (hidden_state,normalized_shape) and node.kwargs {weight, bias, eps}, the different args and kwargs name of lf.layer_norm cause argument error.

I think changing all the argument name of functional.py to nn.functional argment name will fix the similar problem.

**Comment by rachtibat** [comment]
Hey,

are you trying to explain a LLM or ViT?
End of month, I'll update LXT with a much easier to use version for LLMs particularly.

**Comment by Sangyu-Han** [comment]
I am using lxt on ViTs. However, I think this problem is not related with whether it is used for ViT or LLM.
